Resources/Prototypes/_Goobstation/Hydroponics/seeds.yml

```
@@ -0,0 +1,25 @@
+  yield: 4
+  idealLight: 7
+  nutrientConsumption: 0.6
+  chemicals:
+    Artifexium:
+      Min: 2
+      Max: 5
+      PotencyDivisor: 30
+    Vitamin:
+      Min: 1
+      Max: 4
+      PotencyDivisor: 40
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)